resources/views/layouts/minimal.blade.php

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="shortcut icon" type="image/x-icon" href="https://pbs.twimg.com/profile_images/521685439820742657/8B2oQKmP_400x400.jpeg" />
     <meta name="csrf-token" content="{{ csrf_token() }}">
-    <title>@yield('title',__('Welcome')) - ProcessMaker')</title>
```

Comment:
Keep this

Reply:
its still there, just fixed the extra ')'. I can fix it is a different ticket if you would like!

Reply:
- ProcessMaker should always exist untranslated.

Reply:
gotcha!